start

helen-eir opened the pull request.

helen-eir committed:
Add files via upload

helen-eir committed:
Добавила ограничение OS X

helen-eir committed:
Добавила структуру

helen-eir committed:
Update index.html

helen-eir committed:
немного обновила

helen-eir committed:
немного обновила

helen-eir committed:
Дописала разметку

helen-eir committed:
Добавила разметку

helen-eir committed:
Добавила разметку

helen-eir committed:
добавила классы

helen-eir committed:
добавила классы

helen-eir committed:
добавила классы

helen-eir committed:
добавила классы

helen-eir committed:
Update form.html

helen-eir committed:
добавила классы

helen-eir committed:
добавила классы

helen-eir committed:
дописала классы, поправила разметку

helen-eir committed:
поправила разметку

helen-eir committed:
Add files via upload

helen-eir committed:
добавила .jpg и .svg файлы в разметку

helen-eir committed:
добавила .jpg и .svg файлы в разметку

helen-eir committed:
добавила .jpg и .svg файлы в разметку

helen-eir committed:
правка тегов в <form>

helen-eir committed:
Delete catalog_video_desktop.jpg

helen-eir committed:
Delete catalog_video_mobile.jpg

helen-eir committed:
Delete catalog_video_tablet.jpg

romanovaei committed:
начала работу с gulp

romanovaei committed:
начала работу с gulp

helen-eir committed:
Update gulpfile.js

helen-eir committed:
Rename icon_memu_close.svg to icon_menu_close.svg

helen-eir committed:
сделала header, осталось в нем доделать один элемент

helen-eir committed:
сделала header

осталось доделать в header один элемент

helen-eir committed:
доделала header

осталось в header доделать один элемент

helen-eir committed:
оформила header

helen-eir committed:
оформила header

доделать иконку

helen-eir committed:
оформила header

доделать иконку в header

helen-eir committed:
исправила опечатку в названии

romanovaei committed:
работа с less

helen-eir committed:
лишний файл

helen-eir committed:
убрала  border для выравнивания объектов

romanovaei committed:
переписала кнопку меню на jquery

romanovaei committed:
дооформляю блоки section

romanovaei committed:
дооформляю блоки section

romanovaei committed:
дооформляю section, оформила footer

romanovaei committed:
переписала less для index.html под first mobile, еще в процессе работы

helen-eir committed:
главная страница оформлена, в catalog.html и form.html обновила header, footer

модальное окно всплывает при нажатии кнопки "ЗАКАЗАТЬ"

helen-eir committed:
главная страница оформлена

helen-eir committed:
закончила оформление страницы каталога

helen-eir committed:
исправила опечатку

helen-eir committed:
убрала вспомогательный блок

helen-eir committed:
исправила опечатку

helen-eir committed:
Оформляю form.html 

Осталось 2 блока и привести код в порядок

helen-eir committed:
Закончила работу по макету

Далее буду добавлять интерактив и улучшать код

helen-eir committed:
Убрала свойство fill form.html

helen-eir committed:
Еще одно дополнение

helen-eir committed:
удаляю вспомогательный файл

elghst commented on catalog.html:
```diff
@@ -0,0 +1,276 @@
+  <title>Каталог товаров</title>
+</head>
+<body>
+  <header class="header__page container">
```
Тут и ниже элементы header__ не лежат в блоке header
То есть, структура должна быть такая:
```
<div class="header">
    <div class="header__element">
    </div>
    <div class="header__element-2">
        <div class="header__element-3 header__element-3_type_inner">
        </div>
    </div>
</div>
```
Ну с нормальными названиями офк
Полагаю, должно быть
`<header class="header"></header>`

elghst commented on catalog.html:
```diff
@@ -0,0 +1,276 @@
+      <ul class="nav__list">
+
+        <li class="nav__item nav__item_logo">
+          <a class="nav__item_link" href="index.html">
```
`block/element_type_modificator` это синтаксис для модификатора, и его нельзя использовать без блока/элемента, к которому он применяется. Об этом тут: https://ru.bem.info/methodology/naming-convention/
Ну у тебя тут другой случай, наверное, это просто другой элемент? Короче, андерскоры _ в бэме надо использовать только в синтаксисе, а имена блоков и модификаторов из нескольких слов разделять минусами
`<a class="nav__item-link">`

elghst commented on catalog.html:
```diff
@@ -0,0 +1,276 @@
+<body>
+  <header class="header__page container">
+    <nav class="header__nav" id="menu_open">
+      <ul class="nav__list">
```
Тут тоже использование элемента `nav__list` без наличия блока `nav`
Может, ты подумала, что тег `<nav>` это блок? Бэм это только классы

elghst commented on .gitignore:
```diff
@@ -1,5 +1,32 @@
+Temporary Items
+.apdisk
+
+# End of https://www.gitignore.io/api/osx
```
node_modules стоит добавлять в .gitignore любом проекте, чтоб после развертывания локально люди не наблюдали несколько тысяч измененных файлов в гите

helen-eir replied:
Добавила

elghst commented on index.html:
```diff
@@ -0,0 +1,406 @@
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css">
+  <script type="text/javascript" src="js/jquery-3.3.1.min.js"></script>
```
Можно было по cdn подключить, чтоб не запихивать вручную себе в репо
https://cdnjs.com/libraries/jquery/

helen-eir replied:
Изменила

elghst commented on gulpfile.js:
```diff
@@ -0,0 +1,28 @@
+var cleancss = require('gulp-clean-css');
+var browserSync = require('browser-sync');
+
+gulp.task('less', function() {
```
У меня не работает почему-то, если удалить скаченные .css файлы, новые не собирает
Слету не могу подсказать, галпом не пользовался
```
Projects/helen-eir.github.io [master●] » ./node_modules/.bin/gulp less
[07:11:06] Using gulpfile ~/Projects/helen-eir.github.io/gulpfile.js
[07:11:06] Starting 'less'...
[07:11:06] Finished 'less' after 12 ms
Projects/helen-eir.github.io [master●] » 
```

elghst replied:
Вообще, хотел потестировать потому, что в бандле style.css уже оказываются все необходимые стили. Ты рекурсивно каждый .less файл конвертируешь в .css, и все остальные .css файлы грузятся браузером, но лежат без дела, они нигде не используются. Думаю, это стоит убрать

elghst commented on index.html:
```diff
@@ -0,0 +1,406 @@
+</head>
+<body>
+
+  <header class="header__page container">
```
Тут, полагаю, меню одинаковое, второй раз комментировать не буду, оставлю только в catalog.html

elghst commented on catalog.html:
```diff
@@ -0,0 +1,276 @@
+    <nav class="header__nav" id="menu_open">
+      <ul class="nav__list">
+
+        <li class="nav__item nav__item_logo">
```
Сначала указывается тип модификатора (подходящий по смыслу, можно просто  `_type_`), а потом имя мода
`nav__item_type_logo`

elghst replied:
Ну и ниже везде тоже этот случай

elghst commented on catalog.html:
```diff
@@ -0,0 +1,276 @@
+              <img src="img/header_logo_desktop.svg" alt="logo">
+            </picture>
+          </a>
+          <button class="nav__button_menu nav__button_menu_close" type="button"></button>
```
Пустая кнопка?

elghst replied:
Аа, это для мобильной версии
Ну я бы пояснее сделал класс хотя бы, например, `nav__mobile-menu-button nav__mobile-menu-button_type_close`

helen-eir committed:
дополнила

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
ненужный файл

helen-eir committed:
Правки по замечаниям

elghst commented on js/common.js:
```diff
@@ -0,0 +1,22 @@
+$(function () {
+  var openMenu = function() {
```
Что-то странное с отступами
Должно быть как-то так
```
var openMenu = function() {
    $(this).toggleClass("nav__button_menu_close");
    $(this).toggleClass("nav__button_menu_open");
    $(".nav__item_catalog, .nav__item_order, .nav__item_search, .nav__item_cart").toggleClass("nav__item_hidden");
    $(".nav__item_catalog, .nav__item_order, .nav__item_search, .nav__item_cart").toggleClass("nav__menu_open");
};
```

helen-eir replied:
исправила

elghst commented on less/blocks/advantages.less:
```diff
@@ -0,0 +1,84 @@
+@tablet-width: ~"(min-width: 768px)";
```
Убрать в variables.less

elghst commented on less/blocks/advantages.less:
```diff
@@ -0,0 +1,84 @@
+  padding: 0 9.3%;
+}
+
+.advantages__title {
```
```
.advantages {
    &__title {
    }
    &__list {
    }
}
```

elghst commented on less/blocks/advantages.less:
```diff
@@ -0,0 +1,84 @@
+  }
+}
+
+.advantages__item_block {
```
```
&__item {
    &_type {
        &_block {
        }
    }
}
```

helen-eir committed:
работа по БЭМ

helen-eir committed:
работа по БЭМ

helen-eir committed:
работа по БЭМ

helen-eir committed:
работа по БЭМ

helen-eir committed:
работа с БЭМ

helen-eir committed:
работа с БЭМ

helen-eir committed:
Файл попал не туда

helen-eir committed:
Работа с БЭМ

helen-eir committed:
Работа с БЭМ

helen-eir committed:
Работа с БЭМ

romanovaei committed:
delete style.css

romanovaei committed:
delete form__type.less

romanovaei committed:
delete file

romanovaei committed:
delete file

romanovaei committed:
delete files

romanovaei committed:
delete file

helen-eir committed:
Сжатие изображений, добавила Favicon

helen-eir committed:
remove img

helen-eir committed:
remove img

helen-eir committed:
remove img

helen-eir committed:
remove img

helen-eir committed:
remove img

helen-eir committed:
remove img

helen-eir committed:
добавила favicon

helen-eir committed:
Разбила global.less 

Разбила global.less на файлы: global.less, form.less, form-input.less, form-label.less

helen-eir committed:
Разбила global.less

Разбила global.less на файлы: global.less, form.less, form-input.less, form-label.less

helen-eir committed:
Обновила

romanovaei committed:
добавила слайдер для отзывов
